docs/ref/request-response.txt

```diff
@@ -234,6 +234,16 @@ application.
     ``urlconf`` can be set to ``None`` to revert any changes made by previous
     middleware and return to using the :setting:`ROOT_URLCONF`.
 
+.. attribute:: HttpRequest.exception_reporter_filter
+
+    Allows overriding :setting:`DEFAULT_EXCEPTION_REPORTER_FILTER` for the
+    current request. See :ref:`custom-error-reports` for details.
+
+.. attribute:: HttpRequest.exception_reporter_class
+
+    Allows overriding :setting:`DEFAULT_EXCEPTION_REPORTER` for the current
```

Comment:
A small suggestion to keep the same "style" in this section :thinking: :
```
This will be used instead of :setting:`DEFAULT_EXCEPTION_REPORTER` for the current ...
```

Reply:
Yep. Good call. I’ll update.